AVRIoT.X/mcc_generated_files/cloud/wifi_service.h

```diff
@@ -31,7 +31,6 @@ SOFTWARE.
 #include <stdint.h>
 #include <stdbool.h>
 
-#define MAX_WIFI_CRED_LENGTH 31
```

Comment:
Better avoid creating duplicate symbols for the same function.  The proper symbol(s) M2M_MAX_SSID_LEN and M2M_MAX_PASS_LEN are already defined in the winc driver includes m2m/m2m_types.h

Reply:
Thanks Lucio, we will look into these. Appreciate the inout